mapstore/README.md

```diff
@@ -0,0 +1,15 @@
+MapStore configuration files
+----------------------------
+
+- geostore.properties: MapStore specific database configuration properties (not contained in default.properties, e.g. the database schema used by MapStore for its persistence layer)
+- proxy.properties: MapStore proxy configuration (see here: https://github.com/geosolutions-it/http-proxy/wiki/Configuring-Http-Proxy for further details)
+- log4j.properties: logging configuration (see here: https://logging.apache.org/log4j/2.x/manual/configuration.html#ConfigurationSyntax)
+- localConfig.json: main frontend configuration file, in JSON format (see here: https://mapstore.readthedocs.io/en/latest/developer-guide/local-config/)

```

Comment:
Maybe some words here to explain why localConfig.json is not provided ?

Reply:
Hello @fvanderbiest I'm continuing the work done by @Gnafu up to now. I do not understand this question, because I see mapstore/localConfig.json is already provided [here](https://github.com/georchestra/datadir/blob/docker-master/mapstore/localConfig.json)

Reply:
Indeed. Had not seen it in the PR.
Sorry for the noise